app/scripts/GenomePositionSearchBox.js

```diff
@@ -502,7 +502,7 @@ class GenomePositionSearchBox extends React.Component {
     for (let i = 0; i < valueParts.length; i++) {
       if (valueParts[i].length === 0) { continue; }
 
-      const [chr, pos, retPos] = this.searchField.parsePosition(valueParts[i]);
+      const retPos = this.searchField.parsePosition(valueParts[i])[2];
```

Comment:
Isn't this contrary to the "destructure with disabled warning philosophy"?

Reply:
A little: I was more thinking of the cases where we might also be getting `pos` but not `chr`, where it would have to be split over multiple lines if we're not using destructuring... ie, it's not that access by index is bad, but changing one tidy line with an `eslint-disable` into two or more separate assignments feels awkward.

Eh. It depends on context. If you have a strong preference, let me know, but I'm comfortable with this.